addons/account/tests/test_templates_consistency.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from odoo.tests.common import HttpCase
+
+
+class AccountingTestTemplConsistency(HttpCase):
+    '''Test the templates consistency between some objects like account.account when account.account.template.
+    '''
+
+    def check_fields_consistency(self, model_from, model_to, exceptions=[]):
+        '''Check the consistency of fields from one model to another by comparing if all fields
+        in the model_from are present in the model_to.
+        :param model_from: The model to compare.
+        :param model_to: The compared model.
+        :param exceptions: Not copied model's fields.
+        '''
+
+        def get_fields(model, add_domain):
+            # Retrieve fields to compare
+            domain = [('model', '=', model), ('state', '=', 'base'), ('related', '=', False),
+                      ('compute', '=', False), add_domain]
+            return self.env['ir.model.fields'].search(domain)
+
+        from_fields = get_fields(model_from, ('name', 'not in', exceptions))
+        for field in from_fields:
+            child_field = get_fields(model_to, ('name', '=', field.name))
```

Comment:
fetch all fields from model_to only once and put that in a dictionary to avoid making a search() for each field in the loop